codegen/generate_schema.py

```diff
@@ -396,6 +396,7 @@ def generate_dataclass(  # noqa: C901
     name: str,
     fields: Sequence[Field],
     version: int,
+    top_level: bool,
```

Comment:
```suggestion
    top_level: bool = False,
```

nit (non-blocking): Might be nice just to allow omitting it in a few places.

Reply:
Good, applied